src/Control/Effect/Sum.hs

```diff
@@ -27,17 +26,6 @@ instance (Effect l, Effect r) => Effect (l :+: r) where
   handle state handler (R r) = R (handle state handler r)
 
 
--- | Lift algebras for either side of a sum into a single algebra on sums.
---
---   Note that the order of the functions is the opposite of members of the sum. This is more convenient for defining effect handlers as lambdas (especially using @-XLambdaCase@) on the right, enabling better error messaging when using typed holes than would be the case with a binding in a where clause.
-handleSum :: (          sig2  m a -> b)
-          -> ( sig1           m a -> b)
-          -> ((sig1 :+: sig2) m a -> b)
```

Comment:
I’ve opted to delete `handleSum` since pattern matching definitions are more idiomatic. On the other hand, we could take the position that we should leave these in but deprecate them. Thoughts?

Reply:
Killing them is fine given that we’re pre-1.0.0. Once we hit that, we’ll want to pursue less-draconian measures when things disappear.